### PR TITLE
UT coverage publishing  cahnges

### PR DESCRIPTION
**Pull request by mumutyal**

### CODE REVIEW

File: block-storage-attacher/scripts/publishCoverage.sh [hunk 1]
```diff
@@ -17,7 +17,8 @@ GREEN_THRESHOLD=85
 YELLOW_THRESHOLD=50
 
 # clone and prepare gh-pages branch
-git clone -b gh-pages https://$GHE_USER:$GHE_TOKEN@github.ibm.com/$TRAVIS_REPO_SLUG.git .
+#git clone -b gh-pages https://$GHE_USER:$GHE_TOKEN@github.ibm.com/$TRAVIS_REPO_SLUG.git .
```
Comment (akgunjal) [inline block-storage-attacher/scripts/publishCoverage.sh]: Remove this commented line.

File: block-storage-attacher/scripts/publishCoverage.sh [hunk 2]
```diff
@@ -51,12 +52,19 @@ curl https://img.shields.io/badge/Coverage-$NEW_COVERAGE-$BADGE_COLOR.svg > $TRA
 COMMIT_RANGE=(${TRAVIS_COMMIT_RANGE//.../ })
 
 # Generate result message for log and PR
+#if (( $(echo "$OLD_COVERAGE > $NEW_COVERAGE" | bc -l) )); then
+#	RESULT_MESSAGE=":red_circle: Coverage decreased from [$OLD_COVERAGE%](https://pages.github.ibm.com/$TRAVIS_REPO_SLUG/coverage/${COMMIT_RANGE[0]}/cover.html) to [$NEW_COVERAGE%](https://pages.github.ibm.com/$TRAVIS_REPO_SLUG/coverage/${COMMIT_RANGE[1]}/cover.html)"
+#elif (( $(echo "$OLD_COVERAGE == $NEW_COVERAGE" | bc -l) )); then
+#	RESULT_MESSAGE=":thumbsup: Coverage remained same at [$NEW_COVERAGE%](https://pages.github.ibm.com/$TRAVIS_REPO_SLUG/coverage/${COMMIT_RANGE[1]}/cover.html)"
+#else
+#	RESULT_MESSAGE=":thumbsup: Coverage increased from [$OLD_COVERAGE%](https://pages.github.ibm.com/$TRAVIS_REPO_SLUG/coverage/${COMMIT_RANGE[0]}/cover.html) to [$NEW_COVERAGE%](https://pages.github.ibm.com/$TRAVIS_REPO_SLUG/coverage/${COMMIT_RANGE[1]}/cover.html)"
+#fi
```
Comment (akgunjal) [inline block-storage-attacher/scripts/publishCoverage.sh]: Remove all above commented lines.

File: block-storage-attacher/scripts/publishCoverage.sh (hunk 1 above)
Comment (mumutyal) [reply block-storage-attacher/scripts/publishCoverage.sh]: done

File: block-storage-attacher/scripts/publishCoverage.sh (hunk 2 above)
Comment (mumutyal) [reply block-storage-attacher/scripts/publishCoverage.sh]: done

**Pull request merged**